lib/internal/errors.js

```diff
@@ -1104,18 +1105,28 @@ E('ERR_INVALID_PACKAGE_CONFIG', (path, message, hasMessage = true) => {
 }, Error);
 E('ERR_INVALID_PACKAGE_TARGET',
   (pkgPath, key, subpath, target, base = undefined) => {
+    const relError = typeof target === 'string' &&
+      target.length && !StringPrototypeStartsWith(target, './');
     if (key === null) {
       if (subpath !== '') {
         return `Invalid "exports" target ${JSONStringify(target)} defined ` +
         `for '${subpath}' in the package config ${pkgPath} imported from ` +
-        base;
+        `${base}.${relError ? ' - targets must start with "./"' : ''}`;
       } else {
         return `Invalid "exports" main target ${target} defined in the ` +
-          `package config ${pkgPath} imported from ${base}.`;
+          `package config ${pkgPath} imported from ${base}.${relError ?
+            ' - targets must start with "./"' : ''}`;
       }
     } else if (key === '.') {
       return `Invalid "exports" main target ${JSONStringify(target)} defined ` +
-      `in the package config ${pkgPath}${sep}package.json`;
+      `in the package config ${pkgPath}${sep}package.json${relError ?
+        ' - targets must start with "./"' : ''}`;
+    } else if (typeof target === 'string' && target !== '' &&
+      !StringPrototypeStartsWith(target, './')) {
+      return `Invalid "exports" target ${JSONStringify(target)} defined for '${
+        StringPrototypeSlice(key, 0, -subpath.length || key.length)}' in the ` +
+        `package config ${pkgPath}${sep}package.json. ` +
+        '- targets must start with `./`';
```

Comment:
For consistent quoting with previous examples:

```suggestion
        '- targets must start with "./"';
```

That said I know these message themselves have inconsistent double and single quoting...